src/main/java/blackjack/Application.java

```diff
@@ -0,0 +1,58 @@
+package blackjack;
+
+import blackjack.domain.CardDeck;
+import blackjack.domain.Dealer;
+import blackjack.domain.Participants;
+import blackjack.domain.Player;
+import blackjack.view.InputView;
+import blackjack.view.OutputView;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Application {
+
+    private static final String AGREE = "y";
+    private static final String DECLINE = "n";
+
+    public static void main(String[] args) {
+        CardDeck cardDeck = new CardDeck();
+        Dealer dealer = new Dealer();
+        List<Player> players = InputView.inputPlayerNames()
+                                        .stream()
+                                        .map(Player::new)
+                                        .collect(Collectors.toList());
+        Participants participants = Participants.of(dealer, players);
+
+        participants.receiveDefaultCards(cardDeck);
+        OutputView.printDefaultCardMessage(dealer, players);
+        players.forEach(player -> drawMoreCard(player, cardDeck));
+        receiveDealerCard(dealer, cardDeck);
+
+        OutputView.printFinalCardsAndScore(participants);
+        OutputView.printFinalResult(dealer, players);
+    }
+
+    private static void drawMoreCard(Player player, CardDeck cardDeck) {
+        if (!player.isAbleToReceiveCard()) {
+            return;
+        }
+        String answer = InputView.inputAnswerToAdditionalCardQuestion(player);
+        if (AGREE.equals(answer)) {
+            player.receiveCard(cardDeck.draw());
+        }
+        OutputView.printEachPlayerCards(player);
+        if (DECLINE.equals(answer)) {
+            return;
+        }
+        if (player.isAbleToReceiveCard()) {
+            drawMoreCard(player, cardDeck);
+        }
```

Comment:
재귀 보단 while을 이용해도 좋을 것 같아요! 
```suggestion
        while(player.isAbleToReceiveCard()) {
            String answer = InputView.inputAnswerToAdditionalCardQuestion(player);
            if (DECLINE.equals(answer)) {
                return;
            }
            player.receiveCard(cardDeck.draw());
            OutputView.printEachPlayerCards(player);
        }
```

Reply:
indent 1을 유지하려 하다보니 while문을 억지로 바꿨었는데, 그래도 가독성을 고려한다면 while문을 쓰는 게 훨씬 읽기 좋아보이네요:) 피드백 감사합니다!!

Reply:
```
String answer = InputView.inputAnswerToAdditionalCardQuestion(player);
            if (DECLINE.equals(answer)) {
                return;
            }
``` 

이 부분도 메소드 분리해서 while 조건문 안으로 넣으면 indent 1 을 유지할 수 있을 것 같아요 :)